### PR TITLE
query-node: update indexer version

### PR DESCRIPTION
**Pull request by metmirr**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -101,25 +101,28 @@ services:
...
+      - TYPES_JSON=types.json
+      - DB_NAME=${INDEXER_DB_NAME}
+      - DB_HOST=db
+    depends_on:
       - db
-    command: ["workspace", "query-node-root", "indexer:start"] 
+    volumes:
+      - ./types/augment/all/defs.json:/home/hydra/packages/hydra-indexer/types.json
+    command: >
+      sh -c 'yarn db:bootstrap && yarn start:prod'
```
Comment (mnaamani) [inline docker-compose.yml:125]: I don't see `db:bootstrap` in any package.json ?

**Comment by mnaamani** [comment]
The content-directory/query-node integration tests are working :+1:

File: docker-compose.yml (hunk 1 above)
Comment (mnaamani) [reply docker-compose.yml:125]: or is that in indexer container ?

Comment (metmirr) [reply docker-compose.yml:125]: yes, correct! it's in the indexer.

**Pull request merged**